app/replica.go

```diff
@@ -157,6 +157,7 @@ func CloneReplica(s *replica.Server, address string, cloneIP string, snapName st
 	var err error
```

Comment:
is this CloneReplica only for Clone related volumes?

Reply:
yes